sites.json

```diff
@@ -904,6 +904,16 @@
         ]
     },
 
+    {
+        "name": "car2go",
+        "url": "https://www.car2go.com/US/en/",
```

Comment:
This should be a direct link to deletion or, in this case, to the support page where the phone number is listed. Given that there doesn't seem to be a distinct "contact us" page, I'd recommend linking to the contact information on the main page: https://www.car2go.com/US/en/#footer-contactus